setup.cfg

```diff
@@ -1,10 +1,12 @@
 [isort]
 line_length = 120
-known_edx =
+known_third_party = ddt,mock
```

Comment:
I added it because it isort was complaining about them, but that's because I hadn't installed my requirements yet.  Once I did that, it stopped complaining.

isort checks your import sort order, and it considers something a third-party package if it lives in lib/python*/site-packages, so without the packages being installed, it couldn't identify them properly.